hr_attendance_ip_check/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    ip_check_enabled = fields.Boolean(
+        string='Enable IP-based Attendance Check',
+        config_parameter='hr_attendance.ip_check_enabled',
+        help="Enable IP address validation for attendance check-in/check-out"
+    )
+    whitelist_ips = fields.Char(
```

Comment:
Instead of having a comma separated list of IPs which would be very difficult to handle in long run, some other way should be used to handle the IPs.For ex: The allowed IPs can be stored on a per employee basis